assets/js/modules/analytics/datastore/report.js

```diff
@@ -57,7 +57,7 @@ const fetchGetReportStore = createFetchStore( {
 		invariant( isPlainObject( options ), 'Options for Analytics report must be an object.' );
 		invariant( isValidDateRange( options ), 'Either date range or start/end dates must be provided for Analytics report.' );
 
-		const { metrics, dimensions, orderby } = normalizeReportOptions( options );
+		const { metrics, dimensions, dimensionFilters, orderby } = normalizeReportOptions( options );
```

Comment:
I've add a function to normalize dimensions in https://github.com/google/site-kit-wp/pull/2485/commits/a439154ad17f08504a06fb79068d10ff48fe7df8 however not sure what the dimensionFilters possible unnormalized values would be as it's a new param we're introducing. Any thoughts @aaemnnosttv ?

Reply:
The best I could think of is https://github.com/google/site-kit-wp/pull/2485/commits/e623d0317ebea23a001043f817233df3fbe02dd1

Reply:
Re: normalizing `dimensionFilters`, I was mostly referring to the type since I'm not exactly sure what the possible values can be here, but if the types are limited then I think what you have is good 👍